cloudinit/config/cc_landscape.py

```diff
@@ -151,8 +151,17 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
             ]
         )
     )
-    subp.subp(["landscape-config", "--silent"] + cmd_params)
-    util.write_file(LS_DEFAULT_FILE, "RUN=1\n")
+    try:
+        subp.subp(["landscape-config", "--silent", "--is-registered"], rcs=[5])
+        subp.subp(["landscape-config", "--silent"] + cmd_params)
+        util.write_file(LS_DEFAULT_FILE, "RUN=1\n")
```

Comment:
I actually think writing this file a feature that is now obsolete and unnecessary given that we are running landscape-config directly which will start the landscape-client services up and ensure they are active at boot. It was first introduced in 2012 when this module was writing directly to `/etc/landscape/client.conf` and performing a `service landscape-client restart` after writing configs. I don't think the recent landscape-client on any supported Ubuntu series from Focal onward consumes this RUN=1 setting in /etc/default/landscape-client anymore.  So, let's drop the LS_DEFAULT_FILE variable and this write_file  as well.

Reply:
Just validated that all services are appropriately started on Ubuntu focal with landscape-client 18.01-0ubuntu4.3 and no RUN=1 needed in /etc/default/landscape-client. Yes this can be dropped/